release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -122,6 +122,10 @@ For more information, see the link:https://access.redhat.com/articles/7065136[Op
 
 With this release, you can enable bring your own public IPv4 addresses (BYOIP) feature when installing on AWS by using the `publicIpv4Pool` field to allocate Elastic IP addresses (EIPs). You must ensure that you have the xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[required permissions] to enable BYOIP. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
 
+[id="ocp-4-16-installation-and-update-gcp-regions_{context}"]
+==== Deploy GCP in the Dammam, Saudi Arabia and Johannesburg, South Africa regions
```

Comment:
```suggestion
==== Deploy {gcp-short} in the Dammam (Saudi Arabia) and Johannesburg (South Africa) regions
```

Or

Deploy GCP in the Dammam, Saudi Arabia, and Johannesburg, South Africa, regions.

I find the commas more distracting here.